src/pages/en/installation-new.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Installation
+description: How to install Astro with NPM, PNPM, or Yarn.
+---
+## Prerequisites
+
+- **Node.js** - `14.15.0`, `v16.0.0`, or higher.
+- **Text editor** - We recommend [VS Code](https://code.visualstudio.com/) with our [Official Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **Terminal** - Astro is mainly accessed via the terminal’s command-line.
+
+## Install Astro
+
+### Automatic Setup
+
+1. Run one of the following commands in your terminal to start the `create-astro` install wizard.
+
+```shell
+# npm
+npm init astro
+
+# yarn
+yarn create astro
+
+# pnpm
+pnpm create astro
+```
+
+The installation wizard will allow you to
+- Choose from a set of [starter templates](https://github.com/withastro/astro/tree/main/examples) (e.g. basic starter, blog, portfolio, documentation)
+- Select which [additional frameworks](/en/core-concepts/component-hydration) (React, Svelte, Vue, Solid, Preact), if any, you would like to include in your project
```

Comment:
fyi only the `starter` template supports this! Maybe worth mentioning here? 

Ex: You could phrase as "some templates will also ask you to select your favorite component framework" or "you may also be asked to..."

Reply:
Amended!